teams/team-burnham/prod/service.yaml

```diff
@@ -1,10 +1,10 @@
 apiVersion: v1	
 kind: Service	
 metadata:	
-  name: guestbook-ui	
+  name: nginx	
```

Comment:
Minor: we need to do something about those trailing spaces. May be an IDE setting to automatiically remove all of them and reformat?